src/components/SubmitResponsePage.jsx

```diff
@@ -98,23 +82,20 @@ const SubmitResponse = (props) => {
     >
       <Formik>
         {() => {
-          const { type, title, bodyContent, controls } = alertInfo;
+          const { title, bodyContent, controls } = alertInfo;
           return (
             <React.Fragment>
               <div className="clearfix" />{" "}
               {/** Hack to ensure alert displays properly*/}
               <Form>
-                <Alert
-                  className="bc-citysourced-reporter-alert"
-                  type={type}
-                  style={{ marginRight: "20px" }}
-                >
+                <div style={{ marginBottom: "20px" }}>
                   <h2>{title}</h2>
                   {bodyContent}
-                  <div className="cs-form-control">
-                    {controls.map((control) => control)}
-                  </div>
-                </Alert>
+                </div>
+
+                <div className="d-flex justify-content-between">
+                  {controls.map((control) => control)}
```

Comment:
not blocking, but i'm confused by this. It seems like we are passing another react component or list of react components?

Reply:
Its named badly. We are passing in a list of buttons here. A Logout Button or a Return Home button. For some reason we decided that if it failed we wouldnt add the logout button but just the Return Home button thats why its setup like this. Id be fine with just slapping both buttons here instead of doing this map if that makes it easier.

Reply:
That works for me if you want to do that if not, this is resolved.